python/src/puml/helpers/requirements_helpers.py

```diff
@@ -0,0 +1,132 @@
+from typing import List, Optional
+
+
+def _get_requirements_defs(reqs: dict) -> List[dict]:
+    """
+    Helper method for extracting and sorting data from a requirement specification
+
+    Args:
+        reqs (dict): A dictionary of requirement definitions
+
+    Returns:
+        A List of dictionaries of organized data for diagram generation
+    """
+    requirements = []
+    if reqs:
+        for req_key in reqs:
+            req = reqs[req_key]
+            id = req.structure["req"]["id"]
+            attributes = req.structure["req"]["attributes"]
+            type = _get_requirement_type(attributes)
```

Comment:
```suggestion
            tadi_type = _get_requirement_type(attributes)
```
`type` is a protect Python name, so could give some issues.

Reply:
Changed.